src/sections/Statistics/style.module.scss

```diff
@@ -0,0 +1,27 @@
+@use '../../styles/vars.scss' as vars;
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  margin-top: 10px;
```

Comment:
Add flex container to main page and then use gap instead to space out the sections